### DISCUSSION TITLE
New Demo: NodeJS Milestones Timeline

### DISCUSSION THREAD

**Discussion by kumilingus** [clientIO/joint#1992] [News]
This [CodePen](https://codepen.io/jointjs/full/oNMeBGd) shows how take advantage of link to link connections, perpendicular link anchors and custom link end markers to create a beautiful timeline diagram.

<img width="955" alt="image" src="https://user-images.githubusercontent.com/3967880/213072303-8b9f45cf-2ea6-4b7b-a505-3da172fd27c1.png">

**Comment by grafandreas** [comment]
Very nice.
